src/infrastructure/persistance/mongodb/RequestModel.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const requestSchema = new mongoose.Schema({
```

Comment:
you can use class syntax to declare mongoose Model, will be more helpful when you will need to add methods